sw/airborne/subsystems/gps/gps_furuno.c

```diff
@@ -125,4 +125,7 @@ void nmea_parse_perdcrv(void)
   struct LtpDef_i ltp;
   ltp_def_from_ecef_i(&ltp, &gps_nmea.state.ecef_pos);
   ecef_of_ned_vect_i(&gps_nmea.state.ecef_vel, &ltp, &gps_nmea.state.ned_vel);
+
+  /* indicate that a new valid sentence is available */
+  gps_nmea.sentence_available = true;
```

Comment:
msg_available means that the char parser got a whole msg/sentence, sentence_available here means that it was actually a valid/supported sentence and gps info was updated...

Maybe nicer to get rid of sentence_available and make parse_msg return true if valid...